src/Components/About/About.module.scss

```diff
@@ -0,0 +1,205 @@
+@font-face {
+  font-family: "Gilroy";
+  src: url("../font/Gilroy-Bold.ttf");
+  font-weight: bold;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Gilroy";
+  src: url("../font/Gilroy-Medium.ttf");
+  font-weight: medium;
+  font-style: normal;
+}
```

Comment:
Move these font import in the index.scss page and create a new font variable like --srijan4-font-2 (Already done for Gilroy font) etc. and then use it as a variable

```css
font-family: var(--srijan4-font-2);
```